### PR TITLE
feat(editor): Overhaul input selector in NDV

### PR DESCRIPTION
**Pull request by elsmr**
## Summary

- Move input selector
- Add node icon
- Overhaul styling
- Fix duplicate search inputs

image

## Related tickets and issues
https://linear.app/n8n/issue/NODE-1355/overhaul-input-node-selector

## Review / Merge checklist
- [ ] PR title and summary are descriptive. **Remember, the title automatically goes into the changelog. Use `(no-changelog)` otherwise.** ([conventions](https://github.com/n8n-io/n8n/blob/master/.github/pull_request_title_conventions.md))
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included.
   > A bug is not considered fixed, unless a test is added to prevent it from happening again.
   > A feature is not complete without tests.

### CODE REVIEW

File: packages/editor-ui/src/components/RunData.vue [hunk 1]
```diff
@@ -814,6 +836,7 @@ export default defineComponent({
 				defaults.unshift({ label: 'HTML', value: 'html' });
 			}
 
+			console.log(defaults);
```
Comment (OlegIvaniv) [inline packages/editor-ui/src/components/RunData.vue:839]: 👁️

**Review by OlegIvaniv — COMMENTED** [review]
ezgif-4-79a0fb6f75
Works as expected & looks great! Let's just get rid of that one rogue console.log and should be good to go

File: packages/editor-ui/src/components/RunData.vue (hunk 1 above)
Comment (elsmr) [reply packages/editor-ui/src/components/RunData.vue:839]: oops!

**Review by OlegIvaniv — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.45.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.45.0)

**Pull request merged**